Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -543,19 +543,36 @@ bool HiresTexture::LoadTexture(Level& level, const std::vector<u8>& buffer)
 
 std::string HiresTexture::GetTextureDirectory(const std::string& game_id)
 {
-  const std::string texture_directory = File::GetUserPath(D_HIRESTEXTURES_IDX) + game_id;
+  const std::string cstex_altid = game_id.substr(0, 3);  // 3-character region-free version
```

Comment:
What does `cstex_altid` mean? The variable naming is way, way too obscure...

Reply:
custom texture_alternative game identification